builder/docker.go

```diff
@@ -164,20 +165,37 @@ func (b *Builder) getDockerMounts() ([]string, error) {
 // image capable of running the desired command, build that image, and then
 // run the command in that image.
 func (b *Builder) RunCommandInContainer(cmd []string) error {
-	format, err := b.getUpstreamFormat(b.UpstreamVer)
-	if err != nil {
-		return err
+	var imageName string
+	var format string
+	var err error
+	if !Offline {
+		format, err = b.getUpstreamFormat(b.UpstreamVer)
+		if err != nil {
+			return err
+		}
+	} else {
+		if err = helpers.RunCommandSilent("docker", "image", "inspect", "clearlinux/mixer:"+b.State.Mix.Format); err != nil {
+			return errors.Errorf("Failed to find usable docker image, cannot run offline: %s", err)
+		}
+		var hostFormat []byte
+		hostFormat, err = ioutil.ReadFile("/usr/share/defaults/swupd/format")
```

Comment:
Why do you use the host format instead of `b.State.Mix.Format`? When inside the container, the format that that mixer version knows how to build is `b.State.Mix.Format`, not `/usr/share/defaults/swupd/format`

Reply:
hmmm you may be right, unless the user changed their b.State.Mix.Format to a value that is out of range with what image they currently have on their system, but that's the same as their host system being out of range too. I'll update this ^